views/projects/better_egovernment.erb

```diff
@@ -72,9 +72,10 @@
         <div class="row mb-5">
           <div class="col-xs-12">
             <strong>Priebežne aktualizované informácie o projekte a jeho aktivitách</strong>
-            <a href="https://platforma.slovensko.digital/t/lepsie-riesenia-egovernmentu-v-sr-op-evs-projekt/5078">https://platforma.slovensko.digital/t/lepsie-riesenia-egovernmentu-v-sr-op-evs-projekt/5078</a>
-            <p><a href="https://drive.google.com/file/d/1YkeXfjS0QaUByXZsyQbDeVG7VSJD10_2/view">Workshop na tému Rozdeľovanie IT zákaziek na časti</a> (16.4.2019)</p>
-            <p><a href="https://platforma.slovensko.digital/t/seminar-na-temu-pripravne-trhove-konzultacie/6078">Seminár na tému Prípravné trhové konzultácie</a> (19.2.2019)</p>
+            <p><a href="https://platforma.slovensko.digital/t/lepsie-riesenia-egovernmentu-v-sr-op-evs-projekt/5078">https://platforma.slovensko.digital/t/lepsie-riesenia-egovernmentu-v-sr-op-evs-projekt/5078</a><br>
+            <a href="https://drive.google.com/a/slovensko.digital/file/d/1DuRCyD8KIKtLXbAD1QEVKK50jcPnNPJF/view?usp=sharing">Ľudské zdroje vo verejnej správe</a> (Správa z 31.7.2019)<br>
```

Comment:
A preco sme odstranili tie odstavce ked to su rozne projekty?

Reply:
Vyzeralo to dizajnovo cudne tak som Matusovi navrhol toto, co sa mu pacilo

**sucasne:**
![image](https://user-images.githubusercontent.com/44023416/64493883-2f264a80-d286-11e9-9677-73b47967000b.png)

**originalne:**
![image](https://user-images.githubusercontent.com/44023416/64493908-7e6c7b00-d286-11e9-9cf9-ab4261890f57.png)
Napis co si myslis

Reply:
Ok to vyzera lepsie. Mozeme mergovat. Ty mas prava na deploy?